### DISCUSSION TITLE
Replace inpaint area with an object from an input image? Possible?

### DISCUSSION THREAD

**Discussion by retohofstetter** [lllyasviel/Fooocus#1815] [General]
Hi

Inpaint currently does not take any input image and one has to use a text prompt to describe how the inpaint area should be changed. 
Would it be possible to provide an input image that should be put into the inpaint area? 

Say, you want to but a cola can from an input image into a specific position in an image market by the inpaint area. 

Probably this is not possible but I think its worth asking :)

Best, Reto

**Comment by lllyasviel** [comment]
advanced -> dev tool -> control -> mix image prompt and inpainting
